doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,230 @@
+.. _nrf_bm_release_notes_080:
+
+Release Notes for |BMlong| v0.8.0
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|BMshort| v0.8.0 is the public launch release of this product, demonstrating a set of samples for the `nRF54L15 DK`_ with build targets supporting the following SoCs:
+
+* `nRF54L15`_
+* `nRF54L10`_ (emulated on the nRF54L15 DK)
+* `nRF54L05`_ (emulated on the nRF54L15 DK)
+
+The supported DK version is `0.9.3`_.
+
+The release includes a prototype version of the S115 SoftDevice - a proprietary Bluetooth® LE stack to be used on the nRF54L15 DK.
+
+Product maturity
+****************
+
+As this is version 0.8.0 of |BMshort|, please note that the solution is still under development.
+While most modules and libraries are considered stable and are unlikely to undergo significant changes, adjustments might still be necessary.
+This includes potential modifications to the organization of the folder structure.
```

Comment:
Add something about documentation, that we do not have documentation for all modules per now, though it will be added later.

Reply:
"The documentation for |BMshort| is still in development and might not cover all libraries and modules available in the codebase.
Some sections might be incomplete or lacking detailed information as content is being developed."

What do you think ? @eivindj-nordic